pkg/kclient/fakeclient.go

```diff
@@ -2,6 +2,7 @@ package kclient
 
 import (
 	fakeServiceCatalogClientSet "github.com/kubernetes-sigs/service-catalog/pkg/client/clientset_generated/clientset/fake"
+	fake2 "github.com/openshift/odo/pkg/kclient/fake"
```

Comment:
can we call it something more descriptive, like for example `odoFake` ?

Reply:
Done